docs/developer/process/agent-release.md

```diff
@@ -13,7 +13,11 @@ ships a snapshot of [integrations-core][].
 
 ## Setup
 
-Configure your [GitHub](../ddev/configuration.md#github) and [Trello](../ddev/configuration.md#trello) auth.
+Configure your the following and add to your [configuration](../ddev/configuration.md):
+
+- [GitHub](../ddev/configuration.md#github) credentials
+- [Trello](../ddev/configuration.md#trello) credentials
+- Add the `trello_users_$team` table to [automatically assign test cards](#create-items) to team members.
```

Comment:
> `trello_users_$team` table

Should we add/move instructions on how to set these up to the Trello config docs?

Currently the "create items" section reads:

> You must assign each item to a team member after creation and ensure no one is assigned to a change that they authored. [...]

But I think it's fair to make the team mapping a required part of the setup, and just not mention those manual steps anymore.

Then we can say:

```suggestion
- [Trello teams mapping](../dev/configuration.md#trello-teams) (for assigning card items automatically)
```

Don't know if I'm being clear, let me know if this needs clarifying.

Reply:
That makes sense, I can move the trello team mapping as another section in trello configuration.